src/AppBundle/Resources/views/Admin/Client/Client/details.html.twig

```diff
@@ -23,12 +23,14 @@
 {% block helpline %}{% endblock %}
 
 {% block pageContent %}
-
     <p class="flush--bottom">{{ 'client' | trans({}, 'common') }}: <span class="bold">{{ client.fullName }}</span></p>
     {% if client.archivedAt %}
         <p class="flush--bottom">Archived on: <span class="bold">{{ client.archivedAt | date("j M Y") }}</span></p>
     {% endif %}
     <p>{{ 'courtOrderNumber' | trans({}, 'common') }}: <span class="bold">{{ client.caseNumber }}</span></p>
+    {% if client.deletedAt %}
+        <p class="flush--bottom behat-region-discharged-on">Discharged on: <span class="bold">{{ client.deletedAt | date("j M Y") }}</span></p>
```

Comment:
"Discharged on" should be in a translation.

Also, this little definition unit should be [summary list](https://design-system.service.gov.uk/components/summary-list/) rather than a bunch of `<p>`s. I know it's not introduced by your changes, but I think it would be good to change that whilst we're here.

Reply:
Done

Reply:
Nice one 💪